lib/fictium/exporters/api_blueprint_exporter/resource_formatter.rb

```diff
@@ -0,0 +1,31 @@
+module Fictium
+  class ApiBlueprintExporter
+    class ResourceFormatter
+      def format(resource)
+        [build_header(resource), build_actions(resource)]
+          .map(&:strip)
+          .select(&:present?)
+          .join("\n\n")
```

Comment:
:eyes: You often are using `select(&:present?).join("\n\n")`, you can delegate it in BaseExporter